CMSgov/onemac
# Withdraw actions

Story: https://qmacbis.atlassian.net/browse/OY2-16294
Endpoint: https://d2xszls06s64t0.cloudfront.net

### Details
Bug Fix: Withdraw Package Action on Waiver Tab was allowed, but did not function for child components.

### Changes
updated back end to allow withdraw package action to apply to components
updated front end to send the correct component information for the withdraw request.
updated components to know who their parents are
updated seed data to have include children components

### Implementation Notes
Ended up cleaning up some of the updateComponent functionality, making it less aware of the child and more focused on the update

### Test Plan
1. Login as submitter
2. Find a child component that can be withdrawn (you might have to add one using "New Submission" button
3. Click "Withdraw Package" for that child component
4. Deny the action, nothing should change
5. Confirm the action, child should be marked as "Package Withdrawn" in the status.
